scale_build/image/iso.py

```diff
@@ -0,0 +1,94 @@
+import distutils.dir_util
+import glob
+import itertools
+import os
+import shutil
+
+from scale_build.config import VERSION
+from scale_build.utils.manifest import get_manifest
+from scale_build.utils.run import run
+from scale_build.utils.paths import CD_DIR, CD_FILES_DIR, CHROOT_BASEDIR, CONF_GRUB, RELEASE_DIR, TMP_DIR
+
+from .manifest import UPDATE_FILE
+from .utils import run_in_chroot
+
+
+def install_iso_packages(iso_logger):
+    run_in_chroot(['apt', 'update'], logger=iso_logger)
+
+    # echo "/dev/disk/by-label/TRUENAS / iso9660 loop 0 0" > ${CHROOT_BASEDIR}/etc/fstab
+    for package in get_manifest()['iso-packages']:
+        run_in_chroot(['apt', 'install', '-y', package], logger=iso_logger)
+
+    os.makedirs(os.path.join(CHROOT_BASEDIR, 'boot/grub'), exist_ok=True)
+    shutil.copy(CONF_GRUB, os.path.join(CHROOT_BASEDIR, 'boot/grub/grub.cfg'))
+
+
+def make_iso_file(iso_logger):
+    for f in glob.glob(os.path.join(RELEASE_DIR, '*.iso*')):
+        os.unlink(f)
+
+    # Set default PW to root
+    run(fr'chroot {CHROOT_BASEDIR} /bin/bash -c "echo -e \"root\nroot\" | passwd root"', logger=iso_logger, shell=True)
+
+    # Create /etc/version
+    with open(os.path.join(CHROOT_BASEDIR, 'etc/version'), 'w') as f:
+        f.write(VERSION)
+
+    # Copy the CD files
+    distutils.dir_util._path_created = {}
+    distutils.dir_util.copy_tree(CD_FILES_DIR, CHROOT_BASEDIR, preserve_symlinks=True)
+
+    # Create the CD assembly dir
+    shutil.rmtree(CD_DIR, ignore_errors=True)
```

Comment:
What errors are you ignoring?

We expect `CD` to be at least empty after this method terminates successfully. Will it always be so?